### DISCUSSION TITLE
コメント入力用テキストエリアの行間が狭すぎる

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#379] [要望・アイデア・不具合]
コメント入力用テキストエリアの行間が狭すぎるため、テキストエリア内のテキストが窮屈に見えるのと、プレビューしたときの見た目の差が大きすぎる点（プレビューのline-heightは1.9）がちょっと気になりました。
テキストエリアの方ももう少し行間に余裕を持たせた方がいいかもしれません。

![k8LqfS0Xg1](https://user-images.githubusercontent.com/1148320/208227247-5557ecd7-0244-4573-9260-1eac5cad0b0e.gif)

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご意見ありがとうございます！
こちら検討します！💪✨

**Comment by kimkim0814** [comment]
@JunichiIto
お待たせ致しました。
コメント入力用のテキストエリアの行間について、修正が完了しました。
是非お試しいただけると嬉しいです:blush:

**Reply by JunichiIto** [reply]
対応ありがとうございます！よいと思います〜 👍 

![lAdqTmw0ct](https://user-images.githubusercontent.com/1148320/217162953-2c0036d0-1a58-4a93-b490-2619daa397d2.gif)

**Comment by scivola** [comment]
いま見たら行間がとても狭いのですが，戻っちゃいました？　あるいは環境によるのでしょうか。macOS Firefox 121.0 です。

**Reply by JunichiIto** [reply]
そうなんですよねー。
戻ったというか、コメントのデザインが全体的に変わってまた同じような問題を抱えるようになっちゃったのかな？と思ったりしています。
![Seh0kB6zXq](https://github.com/increments/qiita-discussions/assets/1148320/f62a0904-75f7-4a14-8a05-5775d69d1e7a)

**Reply by tanaka-asami415** [reply]
@scivola @JunichiIto 
申し訳ありません🙇💦

調査したところ、コメント入力欄をアップデートしたタイミングでline-heightの指定がなくなっておりました…。
再度修正いたします！

**Reply by tanaka-asami415** [reply]
@scivola @JunichiIto
こちら修正が完了いたしました！
この度はご報告いただきありがとうございます！🙇

**Reply by JunichiIto** [reply]
@tanaka-asami415 修正ありがとうございます！大丈夫そうです〜 🙆‍♂️ 

![x6bcB48zfj](https://github.com/increments/qiita-discussions/assets/1148320/66f45976-06fd-435d-a0cd-cda20f646edb)

**Reply by tanaka-asami415** [reply]
@JunichiIto 
ご確認いただきありがとうございます！🙏✨
